<?xml version="1.0" encoding="UTF-8"?>
<!--
 Generated by Qt.

 Copyright (C) Sapura Secured Technologies, 2014-2020. All Rights Reserved.

 @file
 @version $Id: Call.ui 1376 2020-05-15 06:40:52Z zulzaidi $
-->
<ui version="4.0">
 <class>Call</class>
 <widget class="QWidget" name="Call">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="inpFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="inpLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="callHLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <spacer name="callHSpacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="broadcastButton">
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>76</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>108</width>
            <height>76</height>
           </size>
          </property>
          <property name="text">
           <string>Broadcast</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Images/images/icon_broadcast.png</normaloff>:/Images/images/icon_broadcast.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="callButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>76</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>108</width>
            <height>76</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Images/images/icon_call.png</normaloff>:/Images/images/icon_call.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="callHSpacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
